src/main/resources/i18n/messages.properties

```diff
@@ -642,6 +642,15 @@ tournament.status.openForRegistration=Open
 tournament.status.closedForRegistration=Closed
 tournament.noStartingDate=no starting date set
 tournament.noCompletionDate=not completed yet
+userInfo.foe=This player is a foe
+warn=Attention
```

Comment:
In English, key and value should pretty much be the same. Otherwise, either of it is probably misnamed